common/presenters/events-to-timeline-component.js

```diff
@@ -0,0 +1,65 @@
+const i18n = require('../../config/i18n')
+const addTriggeredEvents = require('../helpers/events/add-triggered-events')
+const setEventDetails = require('../helpers/events/set-event-details')
+
+const getItem = ({
+  component,
+  labelClasses,
+  heading,
+  description,
+  timestamp,
+  byline,
+}) => {
+  return {
+    label: {
+      classes: labelClasses,
+      html: heading,
+    },
+    classes: 'app-timeline__item',
+    html: description,
+    datetime: {
+      timestamp,
+      type: 'datetime',
+    },
+    byline: {
+      html: byline,
+    },
+  }
+}
+
+const eventsToTimelineComponent = (moveEvents, move) => {
+  moveEvents = addTriggeredEvents(moveEvents)
+
+  moveEvents = moveEvents.map(event => setEventDetails(event, move))
```

Comment:
Rather than re-assign this can you move this map to line 35 with the other map.

Reply:
I actually think that make it less clear, but done